src/main/java/me/ryanhamshire/GriefPrevention/Claim.java

```diff
@@ -443,214 +395,184 @@ public boolean hasExplicitPermission(Player player, ClaimPermission level)
         // Check explicit ClaimPermission for UUID
         if (this.hasExplicitPermission(player.getUniqueId(), level)) return true;
 
+        // Special case managers - a separate list is used.
+        if (level == ClaimPermission.Manage)
+        {
+            for (String node : this.managers)
+            {
+                // Ensure valid permission format for permissions - [permission.node]
+                if (node.length() < 3 || node.charAt(0) != '[' || node.charAt(node.length() - 1) != ']') continue;
+                // Check if player has node
+                if (player.hasPermission(node.substring(1, node.length() - 1))) return true;
+            }
+            return false;
+        }
+
         // Check permission-based ClaimPermission
         for (Map.Entry<String, ClaimPermission> stringToPermission : this.playerIDToClaimPermissionMap.entrySet())
         {
             String node = stringToPermission.getKey();
             // Ensure valid permission format for permissions - [permission.node]
-            if (node.length() < 3 || node.charAt(0) != '[' || node.charAt(node.length() - 1) != ']')
-            {
-                continue;
-            }
+            if (node.length() < 3 || node.charAt(0) != '[' || node.charAt(node.length() - 1) != ']') continue;
 
             // Check if level is high enough and player has node
             if (level.isGrantedBy(stringToPermission.getValue())
                     && player.hasPermission(node.substring(1, node.length() - 1)))
-            {
                 return true;
-            }
         }
 
         return false;
     }
 
-    //break permission check
-    public String allowBreak(Player player, Material material)
-    {
-        //if under siege, some blocks will be breakable
-        if (this.siegeData != null || this.doorsOpen)
-        {
-            boolean breakable = false;
+    /**
+     * Check whether or not a player has a certain level of trust for an action.
+     *
+     * @param player the Player performing the action
+     * @param permission the ClaimPermission required to perform the action
+     * @param event the Event resulting in the permission check or null if no event was fired
+     * @return the denial message, or null if the action is allowed
+     */
+    public String checkPermission(Player player, ClaimPermission permission, Event event) {
```

Comment:
I didn't add an overload without an Event for this method specifically because I wanted to suggest to anyone calling it that including the event is highly recommended. Points here:

* Do we want an overload that doesn't require an Event parameter?
* Is the method name sufficiently descriptive?